tools/releaseBuild/azureDevOps/templates/release-ValidateFxdPackage.yml

```diff
@@ -53,9 +68,24 @@ jobs:
       displayName: Expand fxd package
 
     - pwsh: |
+        $env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
+        Import-Module "$(Build.SourcesDirectory)/build.psm1" -Force
+        Find-Dotnet -SetDotnetRoot
+        Write-Verbose -Verbose "DOTNET_ROOT: $env:DOTNET_ROOT"
+        Write-Verbose -Verbose "Check dotnet install"
+        dotnet --info
+        Write-Verbose -Verbose "Start test"
         $packageNameFilter = '${{ parameters.packageNamePattern }}'
         $pwshExeName = if ($packageNameFilter.EndsWith('tar.gz')) { 'pwsh' } else { 'pwsh.exe' }
-        $actualOutput = & $pwshExeName -c 'Start-Job -ScriptBlock { "1" } | Wait-Job | Receive-Job'
+        $pwshPath = Join-Path '$(Pipeline.Workspace)/releasePipeline/finalResults/fxd' $pwshExeName
+
+        if ($IsLinux) {
+          chmod u+x $pwshPath
+        }
```

Comment:
Maybe this is not required  anymore.

Reply:
It is still in master.  We should fix there then backport.

Reply:
Keep this as is since it is this way in master